doc/api/http.md

```diff
@@ -341,13 +347,17 @@ added: v0.7.0
 -->
 
 * `response` {http.IncomingMessage}
-* `socket` {net.Socket}
+* `duplex` {stream.Duplex}
```

Comment:
I would prefer not to rename any arguments, it should be enough to change the type.